ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -761,11 +761,30 @@ public class RouterConfig {
   @Default("false")
   public final boolean routerParanoidDurabilityEnabled;
 
+  /**
+   * This is set in frontendConfig until id converter been fully migrate to router.
+   */
+  public final String idConverterFactory;
+
+  /**
+   * This is set in frontendConfig until id converter been fully migrate to router.
+   */
+  public final String idSigningServiceFactory;
+
+  /**
+   * This is set in frontendConfig until id converter been fully migrate to router.
+   */
+  public final String namedBlobDbFactory;
+
   /**
    * Create a RouterConfig instance.
    * @param verifiableProperties the properties map to refer to.
    */
   public RouterConfig(VerifiableProperties verifiableProperties) {
+    FrontendConfig frontendConfig = new FrontendConfig(verifiableProperties);
```

Comment:
Minor: We could also create `FrontendConfig frontendConfig = new FrontendConfig(verifiableProperties);` in `NonBlockingRouterFactory` and directly use `frontendConfig.idConverterFactory, frontendConfig.idSigningServiceFactory, etc` over there.

Reply:
yeah, I just want the router logic use the router config, and later we can consider remove the idConverterFactory from frontendConfig if id converter has been fully migrate to router layer.